src/test/ui/issues/issue-78957.rs

```diff
@@ -0,0 +1,12 @@
+#![deny(unused_attributes)]
+#![feature(min_const_generics)]
+
+pub struct Foo<#[inline] const N: usize>;
+//~^ ERROR attribute should be applied to function or closure
+pub struct Bar<#[cold] const N: usize>;
+//~^ ERROR attribute should be applied to a function
+//~| WARN this was previously accepted
+pub struct Baz<#[repr(C)] const N: usize>;
+//~^ ERROR attribute should be applied to a struct, enum, or union
+
+fn main() {}
```

Comment:
can you also add the same test for type and lifetime parameters?

Reply:
Fixed.